tasks/interactive-bots.md

```diff
@@ -128,28 +151,38 @@ Follow the tutorial below to create your first simple bot that sends private mes
   ))
   ```
 
-8. Check that the bot is working as expected:
+10. Check that the bot is working as expected:
 
   - it reacts to the `@link` and `@resource` at the beginning of a message
 
-  - it currently sends a private message from the admint account to the author of the original message
+  - it sends a private message to the author of the original message
+
+11. Take screenshots showing that the bot is working, i.a. your terminal window with the bot working, messages sent by the bot to the users. Add the screenshots to `gci/interactive-bots/links/<username>/`.
+
+12. Note down any places you got stuck, problems or errors you ran into while doing this setup process. Add your notes as a `notes.md` file to `gci/interactive-bots/links/<username>/`.
+
+13. Create a commit with the screenshots and notes, with commit message `interactive bots: Run the links bot for <username>.`.
+
+14. Create a pull request in the `zulip/gci-submissions` repository, with title `interactive bots: links bot for <username>`.
+
+*Completion criteria:* Mentors will check that the links bot was properly set up.
 
-[TODO] figure out why some bot credentials don't work locally
+### Task Type C: Create your own bot
 
-## Task 2 - Create your own bots
+Let *feature* be the feature mentioned in the task that brought you here.
 
-Some ideas on what the bots could do:
+1. Make sure you understand what is expected of the feature - what messages should the bot react to, what should it do with the message content and where should it post any output.
 
-  - post to an issue tracker like GitHub
+2. Create your interactive bot at `~/zulip/contrib_bots/lib` as a `.py` file. Make sure it fulfills the requirements.
 
-  - post events to a calendar like Google Calendar
+3. Make sure all the tests pass, using `tools/test-all`.
```

Comment:
What should we recommend for testing here? 

I'd think encouraging students to test is generally good practice and sometimes changes they make might affect tests unexpectedly (e.g. one student was changing lines not needed for his feature and was thus failing linter tests).